templates/main/multipassage.html

```diff
@@ -35,14 +45,14 @@ <h3 class="entry-title text-center">{{object.collections.current.title}}<a id="{
 
 
 {% macro version_menu(objId, inRefs) %}
-
 {% if objects %}
```

Comment:
We should be able to remove these lines here as well since they are now valid for the whole template.

Reply:
It doesn't look like you removed these lines. Does the code not work if these lines are removed?